tests/bluetooth/tester/overlay-bt_ll_sw_split.conf

```diff
@@ -32,7 +32,6 @@ CONFIG_BT_CTLR_ISO_TX_SDU_LEN_MAX=310
 
 # CSIP tests requires up to 3 devices
 CONFIG_BT_MAX_CONN=3
-CONFIG_MBEDTLS_PSA_KEY_SLOT_COUNT=3
```

Comment:
As mention on discord, setting this value higher than `CONFIG_BT_MAX_PAIRED` doesn't really make sense IMO